Package request: Add Sanitizers within the epitest-docker

Hi, it would be nice to provide the library of [Google sanitizers](https://github.com/google/sanitizers) (asan, lsan, ubsan & tsan) with the epitest-docker so that student can compile their project with that library inside of the docker image interactive shell.


Currently compiling a binary that depends on asan & ubsan looks like this:
```
make[1]: Leaving directory '/home/project/tests/compass'
gcc -std=gnu11 -pedantic -Wp,-U_FORTIFY_SOURCE -Wformat=2 -MMD -MP -fanalyzer -fno-builtin -pipe -O2 -march=native -mtune=native -Wall -Wconversion -Wdisabled-optimization -Wduplicated-branches -Wduplicated-cond -Werror=return-type -Wextra -Winit-self -Winline -Wlogical-op -Wmissing-prototypes -Wredundant-decls -Wstrict-prototypes -Wundef -Wunreachable-code -iquote . -fsanitize=address,leak,undefined -g3 --coverage -fprofile-arcs -fprofile-abs-path -Wl,--wrap=write -D wrap_write=__wrap_write -D wrap_write=__wrap_write -D real_write=__real_write .build/tests/hashtable_hash_function.o .build/tests/hashtable_manage.o .build/tests/hashtable_interact.o .build/tests/nano_printf.o .build/tests/put_nbr.o .build/tests/str_utils.o .build/tests/tst_invalid_arguments.o .build/tests/tst_main.o .build/tests/tst_putnbr.o .build/tests/tst_speed.o .build/tests/tst_deletion.o .build/tests/write_mock.o -o unit_tests  -L .build/lib -lcompass
/usr/bin/ld: cannot find /usr/lib64/libasan.so.8.0.0: No such file or directory
/usr/bin/ld: cannot find /usr/lib64/libubsan.so.1.0.0: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:79: unit_tests] Error 1
```

The package names on fedora to install both are `libasan` `libubsan`, (with `lsan` being include by `libasan`)

--- a/Dockerfile
+++ b/Dockerfile
@@ -48,6 +48,9 @@ RUN dnf -y upgrade                          \
         gmp-devel.x86_64                    \
         ksh.x86_64                          \
         langpacks-en                        \
+        libasan.x86_64                      \
+        libubsan.x86_64                     \
+        libtsan.x86_64                      \
         libconfig                           \
         libconfig-devel                     \
         libX11-devel.x86_64                 \

